### PR TITLE
Feature/7277 tests for g1ant ui

### PR DESCRIPTION
**Pull request by danieladamkaiser**

### CODE REVIEW

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.Click/ShouldActivateUIElementCorrectly_WhenClickIsUsed.G1ANT [hunk 1]
```diff
@@ -0,0 +1,7 @@
+addon net version 2.205.17304.1000
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.Click/ShouldActivateUIElementCorrectly_WhenClickIsUsed.G1ANT]: addon net is not needed here

Comment (przegwar) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.Click/ShouldActivateUIElementCorrectly_WhenClickIsUsed.G1ANT]: Remove addon imports from all tests.

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.GetRectangle/ShouldGetBoundsOfWindowCorrectly_WhenGetRectangleIsUsed.G1ANT [hunk 2]
```diff
@@ -0,0 +1,8 @@
+addon ui version 1.0.0.0
+
+program notepad
+ui.getrectangle ‴/ui[@name='Untitled - Notepad']‴ result ♥result
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.GetRectangle/ShouldGetBoundsOfWindowCorrectly_WhenGetRectangleIsUsed.G1ANT]: remove unnecessary result variable

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.SetText/ShouldSetTextCorrectly_WhenSetTextIsUsed.G1ANT [hunk 3]
```diff
@@ -0,0 +1,10 @@
+addon ui version 1.0.0.0
+
+♥inputText = ‴robot‴
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.SetText/ShouldSetTextCorrectly_WhenSetTextIsUsed.G1ANT]: remove ‴‴

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.Click/ShouldActivateUIElementCorrectly_WhenClickIsUsed.G1ANT [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+addon ui version 1.0.0.0
+
+program notepad
+ui.click ‴/ui[@name='Untitled - Notepad']/descendant::ui[@name='Close']‴
+
+test ⊂System.Diagnostics.Process.GetProcessesByName("notepad").Length ==0⊃
```
Comment (przegwar) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Commands/UI.Click/ShouldActivateUIElementCorrectly_WhenClickIsUsed.G1ANT]: Add space between "==0"

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyProduct("G1ANT.Addon.UI.Tests")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("4388d9ec-9c8d-45c1-9204-3d116ac43dbe")]
+
+// [assembly: AssemblyVersion("1.0.*")]
```
Comment (przegwar) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/Properties/AssemblyInfo.cs]: Why this line is commented?

File: G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/G1ANT.Addon.UI.Tests.csproj [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.props'))" />
+    <Error Condition="!Exists('..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.targets'))" />
+  </Target>
+  <Import Project="..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.targets" Condition="Exists('..\..\packages\MSTest.TestAdapter.1.1.11\build\net45\MSTest.TestAdapter.targets')" />
+</Project>
```
Comment (DanielWilczek) [inline G1ANT.Addon.UI/G1ANT.Addon.UI.Tests/G1ANT.Addon.UI.Tests.csproj]: add files to scproj

**Pull request merged**